content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -82,27 +82,184 @@ Costing method
-  the next inventory lot moves up in the queue, an updated product cost is used based on the
-  valuation of that specific lot. This method is arguably the most accurate inventory valuation
-  method for a variety of reasons, however, it is highly sensitive to input data and human error.
+
+.. tabs::
+
+   .. tab:: Standard Price
+
+      The default costing method in Odoo. The cost of the product is manually defined on the product
+      form, and this cost is used to compute the valuation. Even if the purchase price on a purchase
+      order differs, the valuation is the cost defined on the product form.
+
+      .. list-table::
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Operation
+           - Unit Cost
+           - Qty On Hand
+           - Incoming Value
+           - Inventory Value
+         * -
+           - $10
+           - 0
+           -
+           - $0
+         * - Receive 8 products for $10/unit
+           - $10
+           - 8
+           - 8 * $10
+           - $80
+         * - Receive 4 products for $16/unit
+           - $10
+           - 12
+           - 4 * $10
+           - $120
+         * - Deliver 10 products
+           - $10
+           - 2
+           - -10 * $10
+           - $20
+         * - Receive 2 products for $9/unit
+           - $10
+           - 4
+           - 2 * $10
+           - $40
+
+   .. tab:: Average Cost (AVCO)
+
+      Calculates the valuation of a product based on the average cost of that product, divided by
+      the total number of available stock on-hand. With this costing method, inventory valuation is
+      *dynamic*, and constantly adjusts based on the purchase price of products.
+
+      .. list-table::
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Operation
+           - Unit Cost
+           - Qty On Hand
+           - Incoming Value
+           - Inventory Value
+         * -
+           - $0
+           - 0
+           -
+           - $0
+         * - Receive 8 products for $10/unit
+           - $10
+           - 8
+           - 8 * $10
+           - $80
+         * - Receive 4 products for $16/unit
+           - $12
+           - 12
+           - 4 * $16
+           - $144
+         * - Deliver 10 products
+           - $12
+           - 2
+           - -10 * $12
+           - $24
+         * - Receive 2 products for $6/unit
+           - $9
+           - 4
+           - 2 * $6
+           - $36
+
+      .. spoiler:: How are Unit Cost and Inventory Value calculated at each step?
+
+         - When receiving four products for $16 each:
+
+           - **Inventory Value** is calculated by adding the previous inventory value with the
+             incoming value. So, with `$80 + (4 * $16)` = `$80 + $64` = `$144`.
+           - **Unit Cost** is calculated by dividing the inventory value by the quantity on-hand.
+             So, `$144 / 12 = $12`.
```

Comment:
Suggestions to use the :math: role to format the equations. And slightly simplify the sentence and math to a more direct single equation.
```suggestion
        - Inventory value is calculated by adding the previous inventory value with the incoming
          value: :math:`$80 + (4 * $16) = $144`.
        - Unit cost is calculated by dividing the inventory value by the quantity on-hand:
          :math:`$144 / 12 = $12`.
```